data/documentation.json

```diff
@@ -5,14 +5,14 @@
         "beaconId": "0xb5087bcd2a04f5653ad8a6b39fa4fb10fabbdec124b8b489b4d8d610e6392198",
         "name": "CoinGecko BTC/USD 0.1 percent deviation",
         "description": "The public CoinGecko BTC/USD price ticker",
-        "templateUrl": "https://github.com/api3dao/operations/blob/fa18bac5ceb1dbb894fb095a43674fe1d366e125/data/apis/api3/templates/coingecko btc_usd.json",
+        "templateUrl": "https://github.com/api3dao/operations/blob/c69bc1f0a50e36393b31bff46688f0afc847a93c/data/apis/api3/templates/coingecko btc_usd.json",
```

Comment:
I'm guessing this will always be pushed ?

Reply:
Yeah, so `documentation.json` is something for the docs site and probably also the Beacons Dashboard to use. The apps will reference the repository by tag, branch, etc., so pinning the templates to a commit is a safe bet.